index.html

```diff
@@ -552,39 +563,150 @@ <h3>BitstringStatusListEntry</h3>
 includes the `BitstringStatusListCredential` value.
               </td>
             </tr>
+            <tr>
+              <td id="statusSize">statusSize</td>
+              <td>
+The `statusSize` indicates the size of the status entry in bits. `statusSize`
+MAY be provided. If `statusSize` is not present as a property of the
+`credentialStatus`, then `statusSize` MUST be processed as `1`.  `statusSize`
+MUST be an integer greater than zero. If `statusSize` is provided and is greater
+than `1`, then the property `credentialStatus.statusMessage` MUST be present,
+and the number of status messages must equal the number of possible values.
+              </td>
+            </tr>
+            <tr>
+              <td id="statusMessage">statusMessage</td>
+              <td>
+The `statusMessage` property MUST be an array. If present,
+the length of the array must equal the number of possible status states
+indicated by `statusSize`. `statusMessages` MAY be present if
+`statusSize` is `1`. `statusMessages` MUST be present if
+`statusSize` is greater than `1`.  If not present, the message value
+associated with the bit value of `0` is "unset" and the bit
+value of `1` is "set".
+If present, elements in the `statusMessage` array MUST contain at
+minimum two properties:
+                <ul>
+                  <li id="status">
+`status`, a string representing the hexadecimal value of the status prefixed
+with `0x`
+                  </li>
+                  <li id="message">
+`message`, a string used by software developers to assist with debugging
+which SHOULD NOT be displayed to end users.
+                  </li>
+                </ul>
+Implementers MAY add additional values to objects in the `statusMessage` array.
+Implementers MAY use the string value of `undefined` in the value to indicate
+that a corresponding status is not defined for the associated status value, but
+that it may be defined in the future. Rules for how to handle various status
+messages are outside the scope of normative requirements in this document, but
+it is assumed that implementers will document rules for processing various
+status codes.
+              </td>
+            </tr>
+            <tr>
+              <td id="statusReference">statusReference</td>
```

Comment:
Does `statusReference` pose a privacy problem in the same way that uncommitted messages might? Perhaps a hash is also required (e.g., `statusReference: {id: "https://example.com/foo", digestMultibase: "<hash>"}`?

If `statusReference` were committed to via a hash, the `statusMessage` array could potentially be omitted entirely. This might be more attractive than always including all possible messages, now that they must be present in the VC itself. This may also help address the i18n + commitment issue (that I think might exist now as mentioned above) because a more rich response can be returned in the status reference document.

Reply:
could we just treat this identical to an external reference as in the core data model?

Reply:
I think that's the basic idea, it was just being given a more specific relationship name since it's well known how it relates.

Reply:
I thought the `statusReference` was human-readable documentation, and so committing to it using a hash would prevent it being updated in ways that would harm readability?

Again, the design here is strange in that we have a unique value for each message (`statusMessage.status`), but then we have another "developer value" (`statusMessage.message`), that could just as easily live in `statusReference` (if it were a machine-readable file).

It's up to Traceability folks if they want to change the design to something simpler. I'm not going to let that hold up this PR.